scripts/ci/test_plan.py

```diff
@@ -125,6 +125,13 @@ def process(self):
         if not self.platforms:
             self.find_archs()
             self.find_boards()
+        else:
+            for file in self.modified_files:
+                if file.endswith(".rst") or file.endswith(".png") or file.endswith(".jpg"):
```

Comment:
```suggestion
                if file.endswith((".rst", ".png", ".jpg", ".webp")):
```

Reply:
we do not need any of that here actually.